serve/mlc_serve/engine/staging_engine.py

```diff
@@ -174,7 +174,7 @@ def step(self) -> InferenceStepResult:
 
         outputs = list[RequestOutput]()
         with self.requests_lock:
-            LOG.debug("StagingInferenceEngine.step obtained requests_lock", generation_output=generation_output)
+            # LOG.debug("StagingInferenceEngine.step obtained requests_lock", generation_output=generation_output)
```

Comment:
As of https://github.com/octoml/mlc-llm/pull/83, we are forcing [the DEBUG level here](https://github.com/octoml/mlc-llm/blob/6f385cf417622cf53e23d58c86dcd97fee3e40af/serve/mlc_serve/engine/staging_engine.py#L154-L156) and the log has become too noisy because of this line in the hot loop. Is there a way around it? @jroesch

Reply:
There are a lot of debug logging in the server, no much sense to comment the only one place. It's easier to switch to info level and if we want to see on the regular base, modify form debug to info in the server

Reply:
This is the only debug logging in the server which gets printed no matter what the logging level is.

Reply:
@masahi whatever test code you are writing needs to call `logging_utils.configure_logging` in oLLM we do this in the server layer, the workers inherit these settings but if you are running locally you to need to call that first.